poo/abstraction/en/Abstraction.md

```diff
@@ -118,14 +118,14 @@ class Hiking {
 }
 ```
 
-And, henceforth, not only our Hiking class hasn’t access to the Human methods `breathe` and `eat`, since, nowhere, we said to our work with humans (no, but think, if a hiking can make you swallow peas, it would be creepy, still), but more, it accepts the elephants and the tractors !
-Provided they implement `Walking` them too… and, who know how to out of print a tractor.
+And, henceforth, not only doesn't our Hiking class have access to the Human methods `breathe` and `eat`, since, we never specified that it's humans we're working with (think about it, if hiking could make you swallow peas, it would be creepy, right?), but also, it accepts the elephants and the tractors!

```

Comment:
```suggestion
And, henceforth, not only doesn't our Hiking class have access to the Human methods `breathe` and `eat`, since, we never specified that it's humans we're working with (think about it, if hiking could make you swallow peas, it would be creepy, right?), but also it accepts the elephants and the tractors!

```
no coma, classic trap 😄